services/graph/pkg/service/v0/service.go

```diff
@@ -214,29 +225,25 @@ func NewService(opts ...Option) (Graph, error) {
 					r.Get("/sharedWithMe", svc.ListSharedWithMe)
 				})
 			})
-			r.Route("/drives/{driveID}/items/{itemID}", func(r chi.Router) {
-				r.Post("/invite", svc.Invite)
-				r.Route("/permissions", func(r chi.Router) {
-					r.Get("/", svc.ListPermissions)
-					r.Route("/{permissionID}", func(r chi.Router) {
-						r.Delete("/", svc.DeletePermission)
-						r.Patch("/", svc.UpdatePermission)
-						r.Post("/setPassword", svc.SetLinkPassword)
-					})
-				})
-				r.Post("/createLink", svc.CreateLink)
-			})
-
 			r.Route("/drives", func(r chi.Router) {
 				r.Get("/", svc.GetAllDrives(APIVersion_1_Beta_1))
-				r.Route("/{driveID}/items/{itemID}", func(r chi.Router) {
-					r.Post("/invite", svc.Invite)
-					r.Get("/permissions", svc.ListPermissions)
-					r.Delete("/permissions/{permissionID}", svc.DeletePermission)
-					r.Post("/createLink", svc.CreateLink)
+				r.Route("/{driveID}", func(r chi.Router) {
+					r.Post("/root/children", drivesDriveItemApi.CreateDriveItem)
+					r.Route("/items/{itemID}", func(r chi.Router) {
+						r.Delete("/", drivesDriveItemApi.DeleteDriveItem)
+						r.Post("/invite", svc.Invite)
+						r.Route("/permissions", func(r chi.Router) {
+							r.Get("/", svc.ListPermissions)
+							r.Route("/{permissionID}", func(r chi.Router) {
+								r.Delete("/", svc.DeletePermission)
+								r.Patch("/", svc.UpdatePermission)
+								r.Post("/setPassword", svc.SetLinkPassword)
```

Comment:
Shouldn't the endpoint be "password" instead "setPassword". The set is already implied by the http `POST`

Reply:
Maybe. Though I am really hesitant to change that with this PR. It's completely unrelated. I just copied the routes.